src/olympia/versions/tests/test_views.py

```diff
@@ -245,6 +246,7 @@ def test_file_404(self):
         r = self.client.get(reverse('downloads.file', args=[234]))
         assert r.status_code == 404
 
+    @override_settings(MEDIA_URL='http://testserver/media/')
```

Comment:
mmm, didn't this kind of thing used to be defined in settings_test.py?

Reply:
I did that here to not change too many default-settings at once. But I think it makes sense to default on an external URL here since that's what we are using in dev, stage and prod anyway.

Fixed in 2430254465, let's see what Travis thinks about that.